### ISSUE TITLE
Telemetry Competition Mode

### ISSUE THREAD

**Issue by miggy1952** [FrSkyRC/ETHOS-Feedback-Community#4034]
When I turn competition mode on, I am not seeing Rx battery voltage as I would have expected.
When I turn competition mode off, non of the telemetry sensors are restored, I had to rediscover sensors and reset any logic switches that refer to telemetry.
I expected that everything would be restored when I switched on/off

version 1.5.7 EU using ELRS

**Comment by mawzthefinn** [comment]
I'd expect ELRS battery voltage to not work, it's a different sensor ID than the one for ACCST/ACCESS/TANDEM. 

Have you powercycled the transmitter after turning competition mode off? I know that was needed on OTX to prevent somebody from flipping it on and off during competition.

**Comment by miggy1952** [comment]
Yes I tried power cycling but still no joy, all the sensors are lost.

**Comment by agibson2** [comment]
Yea,  The way the competition mode works right now is by also removing any sensors that aren't allowed.  This is problematic as you have to redetect sensors and then go through your setup to re-select the new sensors to any sensors that were removed that you use as a source.  I mentioned this in my other bug report.

https://github.com/FrSkyRC/ETHOS-Feedback-Community/issues/3466

**Comment by agibson2** [comment]
Still hoping something can be done for this.  Needing to redetect sensors and set your config up for those sensors is a bit much.
